### PR TITLE
feat: add three org profile

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

Propose a Skaffold profile to create 3 orgs in standalone mode.
This profile is called "three_orgs".
Sorry about linter....

closes FL-1208

## How has this been tested?

CI + Locally

## Companion

- https://github.com/Substra/substra-backend/pull/733
- https://github.com/Substra/orchestrator/pull/280
- https://github.com/Substra/substra-documentation/pull/365

### CODE REVIEW

**Review by SdgJlbl — APPROVED** [review]
Shall will add linting on yaml files as the frontend is doing? 😂

File: examples/secrets/secret-orchestrator-tls-org-3-client-pair.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  isCA: false
+  privateKey:
+    algorithm: RSA
+    encoding: PKCS1
+    size: 2048
+  usages:
+    - server auth
+    - client auth
+  dnsNames:
+    - substra-backend.org-1.com
```
Comment (thbcmlowk) [inline examples/secrets/secret-orchestrator-tls-org-3-client-pair.yaml:22]: `org-1`? 🤔

Comment (ThibaultFy) [reply examples/secrets/secret-orchestrator-tls-org-3-client-pair.yaml:22]: c'est pareil dans le fichier `tls-org-2)client-pair.yaml` donc j'ai juste dupliquer.... Mais p-e que c'est une erreur dans le fichier org-2 enfaite ?

Comment (thbcmlowk) [reply examples/secrets/secret-orchestrator-tls-org-3-client-pair.yaml:22]: I'm not sure where these values are injected however I don't see why it should be pointing to a different organization (naively).

Comment (ThibaultFy) [reply examples/secrets/secret-orchestrator-tls-org-3-client-pair.yaml:22]: My neither (but even more naively...) @oleobal do you have an opinion on this ?

**Pull request merged**